src/App.js

```diff
@@ -1,15 +1,53 @@
-import React, { useState } from 'react';
+import React, {useState} from 'react';
+import {
+    BrowserRouter as Router,
+    Switch,
+    Route, useHistory
+} from 'react-router-dom';
 import './App.css';
+import TopMenu from './componenten/TopMenu';
+import Blog from './componenten/Blog';
+import Blogpost from './componenten/Blogpost';
+import Login from './componenten/Login';
+
 
 function App() {
-  // We houden in de state bij of iemand is "ingelogd" (simpele versie)
-  const [isAuthenticated, toggleIsAuthenticated ] = useState(false);
-
-  return (
-    <div>
-      Maak hier jouw prachtige blog-applicatie!
-    </div>
-  );
+    // We houden in de state bij of iemand is "ingelogd" (simpele versie)
+    const [isAuthenticated, toggleIsAuthenticated] = useState(true);
+    const history = useHistory();
+
+    function handleClickLogIn() {
```

Comment:
@jamiebloem: je zou een specifiek Private routing-component kunnen maken:

`function PrivateRoute({ children, isAuth, ...rest}) {
  // omdat we nog steeds alle mogelijke properties zoals exact etc. op Route willen zetten, kunnen we met de ...rest operator zeggen:
  // al die andere props die je verder nog ontvangt, zet die ook allemaal maar op <Route>
  return (
    <Route {...rest}>
      {isAuth ? children : <Redirect to="/login" />}
    </Route>
  )
}`